.github/labeler.yml

```diff
@@ -48,3 +48,6 @@
 "YML":
   - any: ["**/*.yml"]
     all: ["!Resources/Maps/_NF/**/*.yml", "!Resources/Prototypes/Maps/_NF/**/*.yml"]
+
+"DB":
```

Comment:
See, the reason I wrote "DATABASE MIGRATIONS" as full words in all-caps is so you wouldn't miss it, and now you've turned it into the tiny and easily overlooked "DB". I don't think this is an improvement at all.

Reply:
Trying to keep it still readable since we do look at tags.